modules/drivers/lidar_surestar/include/component/component_convert.h

```diff
@@ -0,0 +1,132 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http:// www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#ifndef DRIVERS_SURESTAR_INCLUDE_COMPONENT_COMPONENT_CONVERT_H
+#define DRIVERS_SURESTAR_INCLUDE_COMPONENT_COMPONENT_CONVERT_H
+
+#include <pcl/io/pcd_io.h>
+#include <pcl/point_types.h>
+
+#include <memory>
+#include <string>
+#include <thread>
+#include <deque>
+
+#include "cyber/cyber.h"
+#include "modules/drivers/lidar_surestar/include/parser/convert.h"
+#include "modules/drivers/lidar_surestar/proto/sensor_surestar.pb.h"
+#include "modules/drivers/proto/pointcloud.pb.h"
+
+namespace autobot {
+namespace drivers {
+namespace surestar {
+using apollo::cyber::Component;
+class CompSurestarConvert
+    : public Component<adu::common::sensor::Surestar::SurestarScan> {
+ public:
+  bool Init() {
+    // read config file
+    cybertron::proto::SurestarConfig config;
```

Comment:
Maybe `cybertron` should be renamed to `cyber` ?

Reply:
All namespace has been renamed